src/lib/components/cookies/utils/Cookie.js

```diff
@@ -0,0 +1,75 @@
+import Cookies from "js-cookie";
+
+export const stripPII = function (str) {
+    if (!str || str === "") return "";
+
+    return str.replace(/(.*)([A-Z]{1,2}\d{1,2}[A-Z]?\s?\d{1,2}[A-Z]{1,2})(.*)/gi, '$1[REDACTED]$3')
+};
+
+
+export const stripPIIUri = function (str) {
+    if (!str || str === "") return "";
+
+    return str.replace(/(.*)(postcode=[^&]+)(.*)/gi, '$1[REDACTED]$3')
+};
+
+
+export const setCookies = (tag) => {
+    console.info("Cookies accepted - setting up GA.");
+    console.log(`***${tag}`)
+    window[`ga-disable-${tag}`] = false;
+
+    window.dataLayer = window.dataLayer || [];
+
+    function gtag() {
+        window.dataLayer.push(arguments)
+    }
+
+    window.gtag = window.gtag || gtag;
+
+    try {
+        gtag('js', new Date());
+        // GovUK tracker
+        gtag('config', tag, {
+            cookie_domain: 'none'
+        });
+
+
+        console.info("GA successfully set up.", gtag);
+
+    } catch (error) {
+        console.warn("Cookies accepted, but tracking is blocked by the browser.")
```

Comment:
Are these requires? It's not normally ideal for console logging for production

Reply:
We were intending to leave these in until we've tested the analytics in dev.